<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LecteurVue</class>
 <widget class="QMainWindow" name="LecteurVue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LecteurVue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1,0,0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="3">
        <item>
         <widget class="QLabel" name="lTitreDiapo">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>titreDiapo</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lTitreImage">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>titreImage</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1,0,0">
        <item>
         <widget class="QPushButton" name="bPrecedent">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="cartesDisney/image.qrc">
            <normaloff>:/carteDisney/Arrow-PNG-Pic-inverse.png</normaloff>:/carteDisney/Arrow-PNG-Pic-inverse.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="image">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>128</horstretch>
            <verstretch>128</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
          <property name="text">
           <string>image</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="bSuivant">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="cartesDisney/image.qrc">
            <normaloff>:/carteDisney/Arrow-PNG-Pic.png</normaloff>:/carteDisney/Arrow-PNG-Pic.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>30</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0,1">
        <item>
         <widget class="QLabel" name="labelCategorie">
          <property name="text">
           <string>Catégorie</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelPoints">
          <property name="text">
           <string>:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lCategorie">
          <property name="text">
           <string>catégorie</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,0,0,1">
        <item>
         <widget class="QPushButton" name="bLancerDiapo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Lancer diaporama</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lImageActuel">
          <property name="text">
           <string>nbImageActuelle</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="slash">
          <property name="text">
           <string>/</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lImageMax">
          <property name="text">
           <string>nbImageTotale</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bArreterDiapo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Arrêter Diaporama</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPatram_tre">
    <property name="title">
     <string>Paramètre</string>
    </property>
    <addaction name="actionCharger"/>
    <addaction name="separator"/>
    <addaction name="actionVitesse"/>
    <addaction name="separator"/>
    <addaction name="actionEnlever"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionPropos"/>
   </widget>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuPatram_tre"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
  </widget>
  <action name="actionCharger">
   <property name="text">
    <string>Charger diaporama</string>
   </property>
  </action>
  <action name="actionVitesse">
   <property name="text">
    <string>Vitesse de défilement</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionPropos">
   <property name="text">
    <string>À propos de ...</string>
   </property>
  </action>
  <action name="actionEnlever">
   <property name="text">
    <string>Enlever le diaporama</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="cartesDisney/image.qrc"/>
 </resources>
 <connections/>
</ui>
